e2ev2/src/failure-reporter.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs')
+
+class FailureReporter {
+  onTestResult(test, testResult) {
+    // Stole this "isFailed" check from another reporter.
```

Comment:
Was the other reporter in our codebase? If so, is the code common enough that we could have it as utility that we could call from multiple places?

Reply:
No, but we should be on the lookout for common code that can be shared once we start migrating the existing tests.